en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-web-socket-api.md

```diff
@@ -0,0 +1,26 @@
+# Deploy WebSocket API in Choreo Connect
+
+A Web Socket API can be deployed in Choreo Connect via API Manager.
+
+Before You begin
```

Comment:
Before You begin section does not have any steps.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/3767/commits/03ed857c6384c8187453777bd2b9f1d390ca44d4